client/js/router.js

```diff
@@ -63,6 +64,11 @@ const router = new VueRouter({
 			path: "/chan-:id",
 			component: RoutedChat,
 		},
+		{
+			name: "SearchResults",
+			path: "/search/:uuid/:target",
```

Comment:
Can we use `chan-:id/search`? Once [maxleiter/improvedURLs](https://github.com/thelounge/thelounge/tree/maxleiter/improvedURLs) is merged we'll need to change it again, but this makes more sense than throwing the UUID in the address bar, imo.